client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -1,10 +1,11 @@
+/*eslint-disable*/
```

Comment:
👍 You're also welcome to rebase/edit/squash as much as you want on these branches when you work on a branch alone.